SPECS/kernel-dom0/kernel-dom0.spec

```diff
@@ -0,0 +1,1241 @@
+
+%global security_hardening none
+%global sha512hmac bash %{_sourcedir}/sha512hmac-openssl.sh
+%define uname_r %{version}-hvl1.m2
+%ifarch x86_64
+%define arch x86_64
+%define archdir x86
+%define config_source %{SOURCE1}
+%endif
+
+Summary:        Mariner kernel that has dom0 support and works with hypervisor
+Name:           kernel-dom0
+Version:        5.15.34.1
```

Comment:
Is the version of `kernel-dom0` always going to match that of the `kernel`/`kernel-headers` packages? If so, please modify the contents of the `version_matching_groups` variable in `toolkit/scripts/check_entangled_specs.py` to reflect that, so we can use our PR checks to keep these packages in sync.

Reply:
The source and config are not yet finalized, and are not in sync with version of core kernel yet

Reply:
Sounds good, thanks!